projects/mercury/src/metadata/views/facets/DateSelectionFacet.js

```diff
@@ -41,26 +41,37 @@ const DateSelectionFacet = (props: MetadataViewFacetProperties) => {
         if (val.toString() === 'Invalid Date') {
             return false;
         }
-        return (val >= minDate && val <= maxDate);
+        // FNS converts two-digits (XX) input year to 1900+XX. Thus, only 4 digits year are considered as valid.
+        const fourDigitsYearEntered = val.getFullYear().toString().length === 4;
+        return (val >= minDate && val <= maxDate && fourDigitsYearEntered);
     };
 
-    const handleChange = (newValue) => {
-        if (value !== newValue) {
-            setValue(newValue);
-            if (isValid(newValue[0]) && isValid(newValue[1])) {
-                onChange(newValue);
-            } else {
-                onChange(null);
-            }
+    function isValidInterval(startDate, endDate) {
```

Comment:
we don't use `function` syntax in the Mercury code. Please use `const` syntax for consistency.

Reply:
auto-refactoring, fixed the point. 

I would add this eslint rule https://eslint.org/docs/latest/rules/prefer-arrow-callback . It did not work right away, need to play with it a bit. Added comment to https://thehyve.atlassian.net/browse/FAIRSPC-20 not to miss it